webview/src/plots/components/PlotsContainer.tsx

```diff
@@ -137,11 +136,25 @@ export const PlotsContainer: React.FC<PlotsContainerProps> = ({
     </div>
   )
 
+  const isTooltip = (el: Element) => {
+    let currentNode = el
+    while (!['SUMMARY', 'BODY'].includes(currentNode.nodeName)) {
+      if (
+        !!currentNode.className.toLocaleLowerCase &&
+        currentNode.className.toLocaleLowerCase().includes('tooltip')
```

Comment:
With the function checking for just the text `tooltip`, the tooltip icon also doesn't cause the section to toggle: 

https://user-images.githubusercontent.com/43496356/197218952-db72b2c5-ddb9-420c-9af8-ad4ba019c90c.mov

Was this change intended? If so, should we stop showing the mouse pointer when hovering over the icon?

Reply:
It was intended, but I'll remove the pointer.